PanCardView/Common/CardsView.cs

```diff
@@ -2064,5 +2068,22 @@ private void FireItemSwiped(ItemSwipeDirection swipeDirection, int index)
             ItemSwipedCommand?.Execute(args);
             ItemSwiped?.Invoke(this, args);
         }
+
+        protected override void OnPropertyChanged([CallerMemberName] string name = null)
+        {
+            base.OnPropertyChanged(name);
+
+            if (name.IgnoreCaseEquals(nameof(IsVisible)) || name.IgnoreCaseEquals(nameof(Opacity)))
+            {
+                AdjustSlideShow();
+            }
+        }
+
+        private void OnUnloaded(object sender, System.EventArgs e)
+        {
+            AdjustSlideShow(true);
+            Unloaded -= OnUnloaded;
+            Handler?.DisconnectHandler();
```

Comment:
Yeah, it's a very strange decision from the MAUI team. In NET 9 they'll be changing it again so that handlers will auto disconnect by default with the ability to opt out. But for now, we have to call manually as per the [docs](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/handlers/create?view=net-maui-8.0#native-view-cleanup).

> Each platform's handler implementation overrides the DisconnectHandler implementation, which is used to perform native view cleanup such as unsubscribing from events and disposing objects. However, this override is intentionally not invoked by .NET MAUI. Instead, you must invoke it yourself from a suitable location in your app's lifecycle.